nc_time_axis/tests/unit/test_NetCDFTimeDateLocator.py

```diff
@@ -115,5 +115,43 @@ def test_yearly(self):
             self.check(5, 0, 5*365), [31., 485., 942., 1399., 1856.])
 
 
+class Test_tick_values_yr0(unittest.TestCase):
```

Comment:
Given this is quite different from the others, I agree it deserves its own class 👌